### DISCUSSION TITLE
What is the difference between your usage of #tags and [[tags]] in Obsidian?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#12] [Q&A] [answered]
For me the difference between `[[tag]]` and `#tag`  is that `[[tag]]` is what most people use tags for: `[[productivity]]`, `[[business]]`, `[[learning]]`, etc.

the reason I do this is that the files don't exist yet on the graph so the nodes are different colors. and as more notes get *tagged* the size of that node increases, and as I see these nodes "develop" I can tell:

> oh wow, I have a LOT of notes linked to `[[learning]]` 

the link `[[learning]]` links to a file that doesn't exist, but since it has garnered so many connections it's probably worth graduating to being a note in of itself and making it into an emerging MOC. starting to index the connections since there are so many now and letting this organic structure emerge over time instead of imposing a top-down structure from the beginning and rigidly assigning your work to specified categories. allow that organic growth.

for `#tag` i use them solely for indicators of status, which are notes that are inputs needing to be processed, which are seedlings, which are evergreen notes. this way I can do a quick filter to see which of all of my notes in my vault are seedlings: `#🌱️` that need to be fleshed out, linked, and graduated to evergreen notes: `#🌲️` as an example. I hope that succinctly explains the differentiation in my mind between my use of `[[tag]]` and `#tag`

![image](https://user-images.githubusercontent.com/29872822/99897440-56c8bf00-2c4e-11eb-979e-cd0a9c10836d.png)

![image](https://user-images.githubusercontent.com/29872822/99897442-5af4dc80-2c4e-11eb-9aa6-721fbb26b6ca.png)

![image](https://user-images.githubusercontent.com/29872822/99897446-5f20fa00-2c4e-11eb-8ac5-f81c5c828ab4.png)

It could be done with `[[🌱️]]` too, it's just the approach I chose as the `#tag` is a soft link and `[[tag]]` is a hard link, and where ever possible I prefer hard links and the `#🌱️` is just a quick way of seeing a cursory overview of everything where as `[[business]]` is a hard link to every business-related file.

You honestly don't need `#🌱️` at all, that why in roam they're basically the exact same thing. before the graph update, several versions ago before tags appeared on the graph this usage was more poignant, but now it really doesn't matter too much and it's just the approach I've stuck with

![image](https://user-images.githubusercontent.com/29872822/99897460-85df3080-2c4e-11eb-9feb-fa4a88a2bb15.png)

**Comment by tallguyjenks** [comment, marked as answer]
A comprehensive explanation of this entire concept available here in this video:

https://youtu.be/zIh1S7ra3aI

**Comment by bogenschlag** [comment]
I like the “metadata block” at the end of the page (I currently use a folded callout box after the page title in a similar way). But how do you handle inline “tagging” with links in situations where the links are *not* part of the actual text, eg. where with “proper tags” you’d do something like `- Some bullet point #tag1 #tag2`? do you simply add links at the end like `- Some bullet point [[link 1]] [[link 2]]` etc.?

**Comment by krzakx** [comment]
I like it too (as theme styling also) and that was my starting point to use [[tags]].
After downloading your vault template I can't figure out any template for creating random/zettelkasten note? What I do miss. Could you describe specillay how do you get the metadata from bottom?

**Reply by tallguyjenks** [reply]
1. create new note
2. insert template `meta`

that's my usual process, based on the first text character in the title of the note determines what template is applied

**Comment by timrosenberg** [comment]
I like the idea of these tagnotes and I have started implementing them in my vault. I have one question, though: how to you keep track of them? At least with tags there's a list of them that I can open on the sidebar. Do you just rely on autocomplete? Or is there a better way?

**Reply by tallguyjenks** [reply]
when referencing them in new notes i rely on autocomplete
